content/sensu-go/5.16/api/events.md

```diff
@@ -31,8 +31,9 @@ The `/events` API endpoint provides HTTP GET access to [event][1] data.
 The following example demonstrates a request to the `/events` API endpoint, resulting in a JSON array that contains [event definitions][1].
 
 {{< highlight shell >}}
-curl -H "Authorization: Bearer $SENSU_TOKEN" \
-http://127.0.0.1:8080/api/core/v2/namespaces/default/events
+curl -X GET \
+http://127.0.0.1:8080/api/core/v2/namespaces/default/events \
+"Authorization: Bearer $SENSU_TOKEN" \
```

Comment:
Unless this is a github diff thing, I don't believe the last `\` is needed since there isn't anymore to the command on another line. This is true for the next 2 commands below.

Reply:
I think it's because I was copying code from Postman -- it adds a \ after every line

Reply:
that probably explains it. I've never used that myself and can't say why it would be doing that. It might be one of those "it doesn't hurt, but if you any more to the command on a new line and forget the `\`, it will be invalid".

Reply:
I will do one big PR for all the API examples to remove the \ from the last line (I can't remember where I added it anymore, so I will just quickly search through all the API example docs to remove it)
Thank you for mentioning it. It's kind of a problem you wouldn't run into if you build the calls w/Postman, but we can't assume people are doing that with our docs.